persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -113,88 +125,53 @@ public void execute() {
                     + e.getMessage());
             return;
         }
-
+        if (schemaFiles.size() > 1) {
+            errStream.println("ERROR: persist directory contains multiple schema files. ");
+            return;
+        }
         String packageName;
         try {
             packageName = readPackageName(this.sourcePath);
         } catch (BalException e) {
             errStream.println(e.getMessage());
             return;
         }
+        if (module == null) {
+            module = packageName;
+        } else {
+            module = module.replaceAll("\"", "");
+        }
         if (schemaFiles.size() == 0) {
-            schemaFiles.add(packageName);
+            schemaFiles.add(SCHEMA_FILE_NAME);
             try {
-                generateSchemaBalFile(persistDirPath, packageName);
-                errStream.printf("Created model definition file(%s) in persist directory.%n",
-                        packageName + BAL_EXTENTION);
+                generateSchemaBalFile(persistDirPath, schemaFiles.get(0));
+                errStream.printf("Created model definition file(schema.bal) in persist directory.%n");
```

Comment:
```suggestion
                errStream.printf("Created model definition file(model.bal) in the persist directory.%n");
```

Reply:
updated in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)